### PR TITLE
Create list contacts UI

### LINKED ISSUE #4: Create list contacts UI

### PR DESCRIPTION
**Pull request by vijay2136**
fixes #4 
- Added Header and Menubar to contacts page
- Created list of contacts ui
- Added contact delete ui

### CODE REVIEW

File: app/javascript/src/components/Dashboard/Contacts/Avatar.jsx [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import React from "react";
+
+import { Avatar as NeetoAvatar, Typography } from "neetoui";
+
+const Avatar = ({ user }) => {
+  return (
+    <div className="flex items-center space-x-2">
+      <NeetoAvatar size="large" user={user} />
+      <div>
```
Comment (yedhink) [inline app/javascript/src/components/Dashboard/Contacts/Avatar.jsx:9]: Why do we need to make this a div?

File: app/javascript/src/components/Dashboard/Contacts/index.jsx [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+  return (
+    <>
+      <Menubar showMenu={showMenu} title="Contacts" />
+      <Container>
+        <Header title="Contacts" setShowMenu={setShowMenu} />
+        <div className="h-screen w-full space-y-8">
+          <Table
+            className="overflow-hidden odd:bg-gray-100"
+            fixedHeight
```
Comment (yedhink) [inline app/javascript/src/components/Dashboard/Contacts/index.jsx:45]: Implicit true attrs should be at the top of the component's attr list.

**Review by yedhink — COMMENTED** [review]
@vijay2136 

Few comments.

File: app/javascript/src/components/Dashboard/Contacts/Avatar.jsx (hunk 1 above)
Comment (vijay2136) [reply app/javascript/src/components/Dashboard/Contacts/Avatar.jsx:9]: The first div has class flex and second div is inside the first div and its purpose is to group all the typography one after another (class block).

Comment (yedhink) [reply app/javascript/src/components/Dashboard/Contacts/Avatar.jsx:9]: @vijay2136 https://www.loom.com/share/cb2050ec9f9748909251f81d78802912

Comment (vijay2136) [reply app/javascript/src/components/Dashboard/Contacts/Avatar.jsx:9]: https://vimeo.com/668636745/6cd40ea1f0

Comment (yedhink) [reply app/javascript/src/components/Dashboard/Contacts/Avatar.jsx:9]: Got it.

**Pull request merged**